dart/test/mocks.dart

```diff
@@ -91,3 +93,42 @@ final fakeEvent = SentryEvent(
     ),
   ),
 );
+
+class MockPlatformChecker implements PlatformChecker {
```

Comment:
if we decide to go for `Fakes`, I'd name them as `FakePlatformChecker ` and also creating a dart file for it.
or we could `extends Mock` and do the mocking using `mockito`.